src/wovnio/wovnphp/Utils.php

```diff
@@ -53,11 +53,33 @@ public static function isAmp($buffer) {
     return false;
   }
 
-  public static function isFilePathURI($uri) {
+  public static function isFilePathURI($uri, $store) {
     return $uri && (preg_match(self::IMAGE_FILE_PATTERN, $uri) ||
         preg_match(self::AUDIO_FILE_PATTERN, $uri) ||
         preg_match(self::VIDEO_FILE_PATTERN, $uri) ||
-        preg_match(self::DOC_FILE_PATTERN, $uri));
+        preg_match(self::DOC_FILE_PATTERN, $uri) ||
+        self::checkIgnorePaths($uri, $store) ||
+        self::checkIgnoreGlobs($uri, $store));
+  }
+
+  private static function checkIgnorePaths($uri, $store = null) {
+    if (null === $store || !isset($store->settings['ignore_paths']) || isset($store->settings['ignore_paths']) && empty($store->settings['ignore_paths'])) return false;
```

Comment:
The check logic is to hard for reading.
How about implement helper to `Store`?

But this is a little thing, so I approved it.

Reply:
in Store I already implement security check to set array as empty, do you think I could avoid extra checks here and only check if setting is empty only? this means I assume settings is always here

Reply:
If there is like `Store->ignorePaths()`, I think that it's helpful for read and for use.
For example, following codes is easy to read for me.
```
# Store.php
function ignorePaths() {
  if (isset($this->settings['ignore_paths'])) {
    $this->settings['ignore_paths'];
  } else {
    return array();
  }
}

# Utils.php
if (null === $store) {
  return false;
}
foreach($store->ignorePaths() as $path) {
    if (strpos($uri, $path) !== false) {
      return true;
    }
  }
  return false;
}
```